### PR TITLE
AI added

### PR DESCRIPTION
**Pull request by Danzkiz**
This fixes several issues in order to add the AI opponent. Right now the opponent has random moves

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -7,44 +8,61 @@
...
+    create_board()
     while True:
         print_board()
 
         # who's turn is it
         current_player = get_current_player(game_turn)
         print("It is " + current_player + "'s turn\n")
 
-        coordinates = get_coordinates()
+        if no_player == 0:
```
Comment (Ragnar-H) [inline game.py]: so the computer can play against itself?

File: game.py [hunk 2]
```diff
@@ -64,27 +82,34 @@ def get_current_player(num):
...
-    # We know this is a valid input
-    # Lets cast it to Int before we return it
-    x_coord = int(x_coord)
-    y_coord = int(y_coord)
     return [x_coord, y_coord]
 
 
-def is_valid_input(coordinate):
-    if not coordinate.isdigit():
+def is_valid_input(input, min, max):
```
Comment (Ragnar-H) [inline game.py]: nice refactor 👍

File: game.py (hunk 1 above)
Comment (Ragnar-H) [reply game.py]: If you want this to be more human-like you can insert a 3-5 second delay between moves when it is the computers turn.

Comment (Danzkiz) [reply game.py]: Yes, but I use it mainly for testing purposes :)

File: game.py (hunk 2 above)
Comment (Danzkiz) [reply game.py]: Thanks :D

**Pull request merged**